main.py

```diff
@@ -1,7 +1,35 @@
 import init_django_orm  # noqa: F401
 
 from django.db.models import QuerySet
+from db.models import Genre, Actor
 
 
 def main() -> QuerySet:
-    pass
+    genres = ["Western", "Action", "Dramma"]
+    actors = [
+        ("George", "Klooney"),
+        ("Kianu", "Reaves"),
+        ("Scarlett", "Keegan"),
+        ("Will", "Smith"),
+        ("Jaden", "Smith"),
+        ("Scarlett", "Johansson"),
+    ]
+
+    for genre in genres:
+        Genre.objects.create(name=genre)
+    for first, last in actors:
+        Actor.objects.create(
+            first_name=first,
+            last_name=last
+        )
```

Comment:
You are repeating yourself in these lines. Try to use the `for` loop to avoid this. Please, refer to the third point of the checklist for the correct example.